app/views/hiring_staff/schools/_vacancy.html.haml

```diff
@@ -3,5 +3,6 @@
   %td= format_date(vacancy.publish_on)
   %td= format_date(vacancy.expires_on)
   %td= vacancy.status.titleize
+  %td= link_to 'Edit', edit_school_vacancy_path(school, vacancy.id)
```

Comment:
We'll need to add a `colspan=2` attribute to the 'Actions' table heading cell — the border cuts off prematurely.

<img width="465" alt="screen shot 2018-04-03 at 10 37 32" src="https://user-images.githubusercontent.com/822507/38241862-340b3648-372b-11e8-9702-2d7bab10c676.png">